client/.storybook/themeDecorator.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { ThemeProvider } from 'emotion-theming';
+import GlobalStyles from '../src/components/GlobalStyles';
+import daylight from '../src/themes/daylight';
+
+const ThemeDecorator = (storyFn) => (
+  <ThemeProvider theme={daylight}>
+    <GlobalStyles />
```

Comment:
it's better to add the `GlobalStyles` component in the `.storybook/config.js` it doesn't really belong in the ThemeDecorator

Reply:
Ok.